v2/bindings/csharp/SnapPluginLibCs/SnapPluginLib/ServiceCollectionExtensions.cs

```diff
@@ -0,0 +1,12 @@
+﻿using Microsoft.Extensions.DependencyInjection;
+
+namespace SnapPluginLib
+{
+    public static class ServiceCollectionExtensions
+    {
+        public static IServiceCollection AddSnapPlugin(this IServiceCollection services)
```

Comment:
why you need that ?

Reply:
The ServiceCollection Extension Pattern for DI.

Reply:
Example:
- declaration -> The ServiceCollection Extension Pattern
- usage for DI -> https://github.com/solarwinds/api-poller/blob/master/src/api/SolarWinds.Slingshot.ApiPoller.Data.Swis/ServiceCollectionExtension.cs#L14